auth0/resource_auth0_attack_protection.go

```diff
@@ -409,111 +393,169 @@ func updateAttackProtection(ctx context.Context, d *schema.ResourceData, m inter
 }
 
 func deleteAttackProtection(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	api := m.(*management.Management)
+
+	result := multierror.Append(
+		api.AttackProtection.UpdateBreachedPasswordDetection(
+			&management.BreachedPasswordDetection{
+				Enabled: auth0.Bool(false),
+			},
+		),
+		api.AttackProtection.UpdateBruteForceProtection(
+			&management.BruteForceProtection{
+				Enabled: auth0.Bool(false),
+			},
+		),
+		api.AttackProtection.UpdateSuspiciousIPThrottling(
+			&management.SuspiciousIPThrottling{
+				Enabled: auth0.Bool(false),
+			},
+		),
+		api.AttackProtection.UpdateBotDetection(
+			&management.BotDetection{
+				Response: &management.BotDetectionResponse{
+					Policy: auth0.String("off"),
+				},
+			},
```

Comment:
The delete shouldn't just do nothing, but instead actually disable all the AP types. This makes it consistent with all the other deletes e.g. guardian.